content/docs/concepts/security/index.md

```diff
@@ -461,10 +460,23 @@ peers:
   - mtls: {}
 {{< /text >}}
 
-Currently, the mutual TLS setting doesn't require any parameters. Hence,
-`-mtls: {}`, `- mtls:` or `- mtls: null` declarations are treated the same. In
-the future, the mutual TLS setting may carry arguments to provide different
-mutual TLS implementations.
+The mutual TLS setting has an optional `mode` parameter that defines the
+strictness of the peer transport authentication. These modes are documented
+in the [Authentication Policy reference document](/docs/reference/config/istio.authentication.v1alpha1/#MutualTls-Mode).
+
+{{< tip >}}
+The default mutual TLS mode is `STRICT`. Therefore, `mode: STRICT` is equivalent
+to all of the following:
```

Comment:
Hugo seems to insist on putting a unnecessary `</p>` tag right after the colon here, which causes the site to fail linting. Any ideas as to how to fix this?